### DISCUSSION TITLE
Manual/SL-UnityShaderVariables

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#359] [Notes]
# Manual/SL-UnityShaderVariables

https://docs.unity3d.com/Manual/SL-UnityShaderVariables.html

**Comment by docscomments** [comment]
**Comment by `@Baste-RainGames`:**  

_ZBufferParams is listed with "x is (1-far/near)"  
For clarity, it's `1 - (far / near)`. That's what precedence gives, but it's nice if it's actually written out like that so you're sure.

**Comment by R-033** [comment]
`y is 1/cos(spotAngle/4)` is incorrect in the unity_LightAtten section. The correct term is `1/(1 - cos(spotAngle/2))`
